### PR TITLE
add products page

### PR DESCRIPTION
**Pull request by k-e-r**
I added products page. Please review and let me know what you think.

### CODE REVIEW

File: products/index.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>COMP</title>
+    <link rel="stylesheet" href="../shared.css" />
+    <link rel="stylesheet" href="./main.css" />
+    <link
```
Comment (SOunit) [inline products/index.html:10]: maybe font awesome import should be top of css files.

import font awesome css.
shared css can overwrite font awesome.
main css can overwrite shared css and font awesome css.

**Review by SOunit — COMMENTED** [review]
LGTM

File: products/index.html (hunk 1 above)
Comment (SOunit) [reply products/index.html:10]: i don't know the next step when pull request needs fix.
should i reject pull request to get fix?
or you can just fix content of pull request?

Comment (k-e-r) [reply products/index.html:10]: Thank you for your response. When I do "pull request" next time, I'll include this change about "awesome css".

**Pull request merged**